azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/CallCompositeActivity.kt

```diff
@@ -232,8 +234,19 @@ internal class CallCompositeActivity : AppCompatActivity() {
             fragmentClassName
         )
         val transaction: FragmentTransaction = supportFragmentManager.beginTransaction()
+
+        // For accessibility, we are going to turn it off during the transaction
+        // this is because it reads "toggle camera" after the transaction, which isn't really
+        // useful to the user. After the transaction we re-enable it so that the screen reader
+        // works as normal
+        val containerView = findViewById<View>(R.id.azure_communication_ui_fragment_container_view)
+        val oldAccessibilityValue = containerView.importantForAccessibility
+        containerView.importantForAccessibility = View.IMPORTANT_FOR_ACCESSIBILITY_NO_HIDE_DESCENDANTS
         transaction.replace(R.id.azure_communication_ui_fragment_container_view, fragment)
         transaction.commit()
+        transaction.runOnCommit() {
```

Comment:
bracket not needed.  Also, `runOnCommit()` acts more like a commit listener correct?  If so can we move this call to before `commit()`

Reply:
https://developer.android.com/reference/android/app/FragmentTransaction#runOnCommit(java.lang.Runnable)

Reply:
I can probably move the runOnCommit() {} block to before commit, I'll check now. But it does need the brackets. Sorry, misunderstood slightly.

Reply:
Sorry more confusion on my part, I can drop the () but not the {}

Reply:
Pushed

Reply:
yes, I meant `runOnCommit { ..`, don't need the `()` brackets
I see merges but don't see the changes, are you sure you pushed?